pkg/cloud/powervs/assets/deployment.yaml

```diff
@@ -0,0 +1,134 @@
+kind: Deployment
```

Comment:
everywhere we need to change the reference from `ibm-cloud-controller-manager` to `powervs-cloud-controller-manager`